index.html

```diff
@@ -315,5 +315,20 @@ <h1 class="about-title">About <br> Myself</h1>
     </div>
 </article>
 </section>
+    <footer class="foter">
+    <!-- Done by abdul ahad -->
+    <img  class="bg-image" src="./images/contact-form-shapes-mobile@2x.png" alt="">
+    <div class="contact-me">
+        <h1>Contact me</h1>
+        <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+</div>
+<!-- Done by my coding partner -->
+<form action="#">
+    <input type="text" placeholder="Ahad" minlength="1" maxlength="30">
+    <input type="email" placeholder="Ahad@gmail.com" minlength="1" maxlength="30">
+    <textarea class="msg" type="text" placeholder="Write your message here" minlength="1" maxlength="500"></textarea>
```

Comment:
- [ ] Please make sure that all your input fields have the required attribute to activate the validation on submit. Like this 👇️

```
   <input type="text" placeholder="Ahad" minlength="1" maxlength="30" required >
    <input type="email" placeholder="Ahad@gmail.com" minlength="1" maxlength="30" required >
    <textarea class="msg" type="text" placeholder="Write your message here" minlength="1" maxlength="500" required ></textarea>
```